src/installation-info/ck/getCKBaseBundleInstallationInfo.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import type { BundleInstallationInfo } from '../types';
+
+import { isSemanticVersion } from '@/utils/isSemanticVersion';
+import { isCKCdnBaseBundlePackInjected } from '@/cdn/ck/isCKCdnBaseBundlePackInjected';
+
+/**
+ * Returns information about the base CKEditor bundle installation.
+ */
+export function getCKBaseBundleInstallationInfo(): BundleInstallationInfo | null {
+	const { CKEDITOR_VERSION } = window;
+
+	if ( !isSemanticVersion( CKEDITOR_VERSION ) ) {
+		return null;
+	}
+
+	const source = isCKCdnBaseBundlePackInjected( CKEDITOR_VERSION ) ? 'cdn' : 'npm';
```

Comment:
Wouldn't checking if the `CKEDITOR` global exist be enough?

Reply:
I'm not sure if `npm` version sets it in esm mode. If it sets - it'll be enough.